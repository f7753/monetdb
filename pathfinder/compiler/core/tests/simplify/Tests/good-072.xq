for $a in (1,2) for $b in (3,4) return ($a, $b)
