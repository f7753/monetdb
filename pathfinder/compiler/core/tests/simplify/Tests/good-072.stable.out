stdout of test 'good-072` in directory 'compiler/core/tests/simplify` itself:


# 13:01:06 >  
# 13:01:06 >  Mtimeout -timeout 60 ./good-072 good-072 
# 13:01:06 >  

0

# 13:01:06 >  
# 13:01:06 >  Done.
# 13:01:06 >  

